### PR TITLE
Add option to choose a bump strategies for a branch

### PR DESCRIPTION
**Pull request by Sypheos**
## Description

Add an option to choose a bump strategies for a branch

```yml
custom_bump_strategies:
   <branch>: <strategie> # valid strategies are "bump_patch", "bump_rc"
```

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -47,6 +47,8 @@ type Configuration struct {
 
 	Docker   *Docker   `yaml:"docker,omitempty"`
 	Deployer *Deployer `yaml:"deployer,omitempty"`
+
+	NoRC bool `yaml:"noreleasecanditates,omitempty"`
```
Comment (AlexisMontagne) [inline pkg/config/config.go]: what abount a 
```
CustomBumpStrategies map[string]string `yaml:"custom_bump_strategies"`
```

with value like:
```
custom_bump_strategies:
   staging: bump_patch
```
in `pkg/bumper/bumper.go` we can add a global variable like `var bumpStrategies = map[string]func(*version.Version){ ... }`

**Review by AlexisMontagne — APPROVED** [review]

**Comment by jlevesy** [comment]
What's the need ? Why do we want to do this ?

File: pkg/config/config.go [hunk 2]
```diff
@@ -47,6 +47,8 @@ type Configuration struct {
 
 	Docker   *Docker   `yaml:"docker,omitempty"`
 	Deployer *Deployer `yaml:"deployer,omitempty"`
+
+	CustomBumpStrategies map[string]string `yaml:"custom_bump_strategies,omitempty"`
```
Comment (jlevesy) [inline pkg/config/config.go]:
```suggestion
	BumpStrategies map[string]string `yaml:"bump_strategies,omitempty"`
```

Custom doesn't bring much here ?

**Review by jlevesy — APPROVED** [review]
Nit, otherwise LGTM

**Comment by Sypheos** [comment]
> What's the need ? Why do we want to do this ?

One of the tool I need to use for a project do not handle `-rcX` releases.

**Comment by AlexisMontagne** [comment]
@Sypheos  If you change the name of the key, change the readme as well

File: README.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ deployer:
   envs:
     key: val
     ...
+custom_bump_strategies:
+   <branch>: <strategie> # valid strategies are "bump_patch", "bump_rc"
```
Comment (tgallice) [inline README.md]:
```suggestion
   <branch>: <strategy> # valid strategies are "bump_patch", "bump_rc"
```
?

**Review by AlexisMontagne — APPROVED** [review]

**Pull request merged**